driver-mapping/src/main/java/com/datastax/driver/mapping/config/DefaultPropertyTransienceStrategy.java

```diff
@@ -0,0 +1,111 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.mapping.config;
+
+import com.datastax.driver.mapping.AnnotatedMappedProperty;
+import com.datastax.driver.mapping.MappedProperty;
+import com.datastax.driver.mapping.annotations.*;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * A {@link PropertyTransienceStrategy} that adopts a permissive, opt-out approach that
+ * will consider a property to be non-transient by default, unless:
+ * <ol>
+ * <li>The property is annotated with {@link com.datastax.driver.mapping.annotations.Transient @Transient};</li>
+ * <li>The corresponding field is non-null and is marked with the keyword {@code transient};</li>
+ * <li>The property name has been explicitly black-listed.</li>
```

Comment:
Its name was passed to the constructor. This comment needs to explain it in more detail.